src/profiles.js

```diff
@@ -5107,6 +5107,8 @@ const profiles = {
     Person: {
         service: './src/services/person/person.service.js',
         versions: [VERSIONS['4_0_0']],
+        filterByPerson: true,
+        filterBy: 'link.target.reference',
```

Comment:
Can it do that filtering recursively? So if Person A linked to B linked to C, then A would be able to query for and get C?